msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/MessageForumStatisticsBean.java

```diff
@@ -2184,6 +2185,21 @@ private String getExternalParameterByKey(String parameterId)
 		return (String) paramMap.get(parameterId);    
 	}
 
+	public boolean getIsAuthor() {
+		String siteUserId;
+		String currentUserId;
+		try
+		{
+			siteUserId = userDirectoryService.getUser(selectedSiteUserId).getId();
```

Comment:
I think there's something wrong in here... If the 'selectedSiteUserId' is the eid then you should call getUserByEid, and if it's not then you can directly use that id in the comparison.